### PR TITLE
Rake docker:build task uses docker compose subcommand

### PR DESCRIPTION
**Pull request by danidoni**
On systems with more recent docker versions, docker-compose may not exist as docker has a compose subcommand now.

### CODE REVIEW

**Comment by danidoni** [comment]
> Wouldn't it be easier to wholesale move over to `docker compose`? this was added a couple of years back and should be supported with any supported docker version

Hm, probably yes... let me change it

**Comment by eduardoj** [comment]
I also found ocurrences of the `docker-compose` string in the following files: `README.md`, `CONTRIBUTING.md` and `src/api/lib/tasks/dev/minitest.rake`. I guess we should also change those ocurrences.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -219,11 +219,11 @@ Windows.
 that require an case sensitive filesystem (default in Linux, configurable in **MacOS/Windows**),
 make sure you run all this from a filesystem that supports this. Here you have [some instructions](https://github.com/openSUSE/open-build-service/wiki/Setup-an-OBS-Development-Environment-on-macOS) in case you are a MacOS user.
 
-1. Install [docker](https://www.docker.com) and [docker-compose (version >= 1.20.0)¹](https://docs.docker.com/compose/).
+1. Install [docker](https://www.docker.com)
    There is documentation about this for [openSUSE](https://en.opensuse.org/SDB:Docker) and various
    [other operating systems](https://docs.docker.com/engine/installation/).
 
-   ¹ A version equal to or greater than _1.20.0_ is required for _docker-compose_ as we depend on the
+   ¹ A greater than _2.00_ is required for _docker compose_ subcommand
```
Comment (eduardoj) [inline CONTRIBUTING.md:226]:
```suggestion
   ¹ A version greater than _2.00_ is required for _docker compose_ subcommand 
```

Comment (danidoni) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -219,12 +219,11 @@ Windows.
 that require an case sensitive filesystem (default in Linux, configurable in **MacOS/Windows**),
 make sure you run all this from a filesystem that supports this. Here you have [some instructions](https://github.com/openSUSE/open-build-service/wiki/Setup-an-OBS-Development-Environment-on-macOS) in case you are a MacOS user.
 
-1. Install [docker](https://www.docker.com) and [docker-compose (version >= 1.20.0)¹](https://docs.docker.com/compose/).
+1. Install [docker](https://www.docker.com)
```
Comment (eduardoj) [inline CONTRIBUTING.md:222]:
```suggestion
1. Install [docker (version >= v2)¹](https://www.docker.com)
```
We should not forget the reference described with the `¹`.

And also, shouldn't be the minimum required version  `v2`?

Comment (danidoni) [reply CONTRIBUTING.md:222]: Yes, it should be

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -219,12 +219,11 @@ Windows.
...
 make sure you run all this from a filesystem that supports this. Here you have [some instructions](https://github.com/openSUSE/open-build-service/wiki/Setup-an-OBS-Development-Environment-on-macOS) in case you are a MacOS user.
 
-1. Install [docker](https://www.docker.com) and [docker-compose (version >= 1.20.0)¹](https://docs.docker.com/compose/).
+1. Install [docker (version >= v2)¹](https://www.docker.com)
    There is documentation about this for [openSUSE](https://en.opensuse.org/SDB:Docker) and various
    [other operating systems](https://docs.docker.com/engine/installation/).
 
-   ¹ A version equal to or greater than _1.20.0_ is required for _docker-compose_ as we depend on the
-   `--use-aliases` flag for the command `docker-compose run` in our development environment.
+   ¹ A version greater than _2.00_ is required for _docker compose_ subcommand to work.
```
Comment (eduardoj) [inline CONTRIBUTING.md:226]:
```suggestion
   ¹ A version greater than _v2_ is required for _docker compose_ subcommand to work.
```
Only for consistency...

**Review by eduardoj — APPROVED** [review]

**Pull request merged**